Sources/MapboxNavigation/NavigationViewportDataSource.swift

```diff
@@ -204,7 +204,7 @@ public class NavigationViewportDataSource: ViewportDataSource {
             
             // Bottom of the viewport on CarPlay should be placed at the same level with
             // trip estimate view.
-            carPlayCameraPadding.bottom += 25.0
+            carPlayCameraPadding.bottom += 70.0
```

Comment:
Agree, it's pretty tricky though. Mostly because `NavigationViewportDataSource` lives independently and doesn't know much about UI changes applied on CarPlay. Will have to investigate possible approaches.